src/app/rewards/page.tsx

```diff
@@ -0,0 +1,89 @@
+"use client";
+
+import { Rewards } from "components/Rewards";
+import { useEffect } from "react";
+import { initializeSubscriptions, unsubscribeAll } from "../subscriptions";
+import { store } from "../state/store";
+
+export default function Markets() {
+  useEffect(() => {
+    initializeSubscriptions(store);
```

Comment:
This probably should not happen here (and inside other pages), but instead somewhere in `src/app/layout.tsx` 🤔  will it init subscriptions twice if we navigate to another page? Should it?

Reply:
Added PR #368 to fix this.